<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enseignant</class>
 <widget class="QDialog" name="enseignant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Last Name : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lastname_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Mail : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mail_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Num_Insc : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="numinsc_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>140</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchbar"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="add_pushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_pushButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modify_pushButton">
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_pushButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>lastname_lineEdit</tabstop>
  <tabstop>mail_lineEdit</tabstop>
  <tabstop>numinsc_lineEdit</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>add_pushButton</tabstop>
  <tabstop>clear_pushButton</tabstop>
  <tabstop>modify_pushButton</tabstop>
  <tabstop>delete_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
